rtl/include/eclhelper.hpp

```diff
@@ -2394,6 +2395,9 @@ struct IHThorDiskReadBaseArg : extends IHThorCompoundBaseArg
     virtual unsigned getProjectedFormatCrc() = 0;
     virtual void getEncryptKey(size32_t & keyLen, void * & key) = 0;
     virtual void setCallback(IThorDiskCallback * callback) = 0;
+    virtual const char * queryFormat() = 0;
```

Comment:
This requires a change to the eclhelper version, I think (as other classes are derived from this)

Reply:
yes, and it is not possible to make it backward compatible.  So potentially problematic.